unit/json/json_parser.cpp

```diff
@@ -133,3 +133,110 @@ SCENARIO("Loading JSON files")
     }
   }
 }
+
+TEST_CASE("json equality", "[core][test-gen-util][state][type]")
```

Comment:
I actually don't know what those [...][...] parts are good for at all, but this particular entry ("test-gen-util") strikes me as information of no value in this repository.